projects/neural_correlations/EXP1-Random/NeuCorr_Exp1.ipynb

```diff
@@ -502,18 +561,27 @@
    },
    "outputs": [],
    "source": [
-    "# Now let's do the same experiment but with all 2^10 binary strings\n",
+    "trials = 10\n",
```

Comment:
I doubt 10 repeats is enough. I found that it is extremely hard to find 3-spike patterns with random sampling. It may take thousands of repeats before you can get one 3-spike pattern, let along even high-order patterns